rpa-builder/6.1/modules/ROOT/pages/toolbox-variable-handling-activity-parameters.adoc

```diff
@@ -10,6 +10,22 @@ image:toolbox-activity-parameters.png[The Activity Parameters Action Step, 40%,
 
 Only one _Activity Parameters_ Action Step can exist in a process and it is automatically placed in the _Workflow Initialization_ section when creating a new Workflow.
 
+Activity Parameters consist of a name, a description, and a value. Activity parameter names have the following requirements:
+
+* The name consists of a combination of letters, numbers, and underscores.
+* The name contains no special characters or spaces.
+* The initial letter of the name is lower case.
+* The name is unique.
++
+You cannot use the same name multiple times for different variables.
+
+The predefined values are test values that RPA Builder uses only during a local run. The variables register automatically on RPA Manager when uploaded. RPA Manager enables you to modify the values of the variables and define them differently, depending on the station on which the Workflow is being executed. See xref:advanced-concepts-using-variables.adoc[Using Variables] for additional information.
+
+Please note that the test values may change in the course of the business process due

```

Comment:
The test values might change in the course of the business process due

```

Reply:
I would get rid of "Please note that." if you want to keep it, my suggestion is to remove "Please."